NN_2L2H2O_poly2d/Makefile

```diff
@@ -22,7 +22,7 @@ NVCC           := $(CUDA_PATH)/bin/nvcc -ccbin $(HOST_COMPILER)
 # flags
 CCFLAGS   := -std=c++11
 NVCCFLAGS := -Wno-deprecated-gpu-targets
-LDFLAGS   := -lcudnn -lcublas -lhdf5 -lhdf5_cpp
+LDFLAGS   := -lcudnn -lcublas -lhdf5 -lhdf5_cpp -O2
```

Comment:
Hello,
I tested with O3. The improvement is quite limited (around 0.5% improvement vs O2).
I leave the optimization as O3 now.

Regards,
Yaoguang